compiler/luci-interpreter/src/kernels/Abs.cpp

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "kernels/Abs.h"
+
+#include "kernels/Utils.h"
+
+namespace luci_interpreter
+{
+namespace kernels
+{
+
+Abs::Abs(const Tensor *input, Tensor *output) : Kernel({input}, {output}) {}
+
+void Abs::configure()
+{
+  LUCI_INTERPRETER_CHECK(input()->element_type() == output()->element_type());
+
+  output()->resize(input()->shape());
+}
+
+void Abs::execute() const
+{
+  switch (input()->element_type())
+  {
+    case DataType::FLOAT32:
+      eval<float>();
+      break;
+    case DataType::S32:
+      eval<int>();
+      break;
+    case DataType::S8:
+      eval<int8_t>();
+      break;
+    case DataType::U8:
+      eval<uint8_t>();
+      break;
+    case DataType::S16:
+      eval<int16_t>();
+      break;
```

Comment:
There are some issues in integer types.

- s32: Size of `int` differs across platforms.
- u8, s8, s16: Quantized value. These types should be handled differently.

Please remove them and only leave FLOAT32. FLOAT32 is enough for quantization.

Reply:
done